src/components/NavBar/SearchBarDropdown.tsx

```diff
@@ -14,13 +14,15 @@ import { Box } from 'nft/components/Box'
 import { Column, Row } from 'nft/components/Flex'
 import { subheadSmall } from 'nft/css/common.css'
 import { GenieCollection, TrendingCollection } from 'nft/types'
-import { ReactNode, useEffect, useMemo, useState } from 'react'
+import React, { useEffect, useMemo, useState } from 'react'
```

Comment:
In general, try to only import what you're using. In this case, you shouldn't be importing `React`, as you'll only ever use its properties.

This doesn't matter so much for React because it's only distributed as CommonJS (cjs), but for ESModules (esm) importing the global will unnecessarily increase our bundle size, so it's a good habit to follow.